packages/crab-netting/test/Netting.t.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import {ERC20} from "openzeppelin/token/ERC20/ERC20.sol";
+
+import {CrabNetting} from "../src/CrabNetting.sol";
+
+contract FixedERC20 is ERC20 {
+    constructor(uint256 initialSupply) ERC20("USDC", "USDC") {
+        _mint(msg.sender, initialSupply);
+    }
+}
+
+contract NettingTest is Test {
+    FixedERC20 usdc;
+    FixedERC20 crab;
+    CrabNetting netting;
+
+    uint256 internal ownerPrivateKey;
+    address internal owner;
+    uint256 internal depositorPk;
+    address internal depositor;
+    uint256 internal withdrawerPk;
+    address internal withdrawer;
+
+    function setUp() public {
+        usdc = new FixedERC20(10000 * 1e18);
+        crab = new FixedERC20(10000 * 1e18);
+        netting = new CrabNetting(address(usdc), address(crab));
+
+        ownerPrivateKey = 0xA11CE;
+        owner = vm.addr(ownerPrivateKey);
+
+        depositorPk = 0xA11CA;
+        depositor = vm.addr(depositorPk);
+
+        withdrawerPk = 0xA11CB;
+        withdrawer = vm.addr(withdrawerPk);
+
+        usdc.transfer(depositor, 400 * 1e18);
+        crab.transfer(withdrawer, 40 * 1e18);
+
+        vm.startPrank(depositor);
+        usdc.approve(address(netting), 200 * 1e18);
+        netting.depositUSDC(20 * 1e18);
+        netting.depositUSDC(100 * 1e18);
+        netting.depositUSDC(80 * 1e18);
+        assertEq(netting.usd_balance(depositor), 200e18);
+        vm.stopPrank();
+
+        vm.startPrank(withdrawer);
+        crab.approve(address(netting), 200 * 1e18);
+        netting.depositCrab(5 * 1e18);
+        netting.depositCrab(4 * 1e18);
+        netting.depositCrab(11 * 1e18);
+        assertEq(netting.crab_balance(withdrawer), 20e18);
```

Comment:
Those deposit calls should be done inside the test function for the related case, e.g inside `testNetting()` and `testNettingWithMultipleDeposits()`

Reply:
Also I think worth adding a simple test for the actual `depositCrab()` and `depositUSDC()`